contracts/RewardsDistributorDelegate.sol

```diff
@@ -435,6 +469,38 @@ contract RewardsDistributorDelegate is RewardsDistributorDelegateStorageV1, Expo
         setCompBorrowSpeedInternal(cToken, compSpeed);
     }
 
+    /**
+     * @notice Set borrower PCT
+     */
+    function _setBorrowerPCT(uint256 _borrowerPCT) public {
+        require(msg.sender == admin, "only admin can set comp speed");
+        borrowerPCT = _borrowerPCT;
+        supplierPCT = TOTAL_PCT.sub(_borrowerPCT);
+    }
+
+    /**
+     * @notice Set new borrow / supply speed
+     * @param cToken The market whose COMP speed to update
+     */
+    function _updateSpeedWithNewEpoch(CToken cToken) public {
+        require(block.timestamp.sub(startTime) >= WEEK, "Must be at least week since latest epoch");
```

Comment:
potential issues:

#1: if _updateSpeedWithNewEpoch not called from cron job w fast enough cadence it could be spreading rewards for longer duration than the speed duration was set for (longer than a week). But should not be a problem as long as we continue minting each week so even if we call late they can claim from balance of subsequent (essentially kick can down road indefinitely)

#2: if AVG_BLOCKS_PER_WEEK is off, we could be giving out more rewards than speed duration set for (ex: we assume AVG is x but we end up having 1.0025x blocks so we spread out same rewards for more blocks. But should average out and again should not be a problem as long as we continue minting each week so even if we call late they can claim from balance of subsequent (essentially kick can down road indefinitely).